content/en/lotus/manage/chain-management.md

```diff
@@ -74,6 +74,20 @@ lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws
 
 {{< /alert >}}
 
+#### New Lightweight Snapshot Service
+
+We have soft launched a new Lightweight chain snapshot service which will be replacing the snapshots above in the future. More information about these snapshots can be found in [Notion](https://pl-strflt.notion.site/Lightweight-Filecoin-Chain-Snapshots-17e4c386f35c44548f5863afb7b5e024).
+
+**Mainnet**
+```shell
+lotus daemon --import-snapshot https://snapshots.mainnet.filops.net/minimal/latest
+```
+
+**Calibrationnet**
+```shell
+lotus daemon --import-snapshot https://snapshots.calibrationnet.filops.net/minimal/latest
+```
```

Comment:
````suggestion
**Mainnet**
```shell
curl -sL https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car.zst | zstd -dc | lotus daemon --import-snapshot /dev/stdin 2>&1 | tee -a ~/daemon-run.log 
```

**Calibrationnet**
```shell
curl -sL https://fil-chain-snapshots-fallback.s3.amazonaws.com/calibrationnet/minimal_finality_stateroots_latest.car.zst | zstd -dc | lotus daemon --import-snapshot /dev/stdin 2>&1 | tee -a ~/daemon-run.log 
```
````

How's this @ribasushi? Haven't tested this though... On Windows right now.

Reply:
@johnnymatthews this is a new service with different import urls that do not support zst at the moment. @ribasushi comment isn't about this PR, but about supporting zstd in the new serivce. I've filed an issue for it over in the project repository. https://github.com/filecoin-project/filecoin-chain-archiver/issues/40

Reply:
Ah gotcha. I'll resolve this thread then.